src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
 export interface Figure {
-
+  shape: string,
+  color: string,
+  getArea(): number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const MaxValue = Math.max(a, b, c);
+
+    if (a <= 0 || b <= 0 || c <= 0 || MaxValue >= (a + b + c) - MaxValue) {
+      throw new Error('Invalid Values');
```

Comment:
Check 3 point in the [checklist](https://github.com/mate-academy/ts_figures_classes/blob/master/checklist.md)